packages/netsuite-adapter/src/change_validators/client_validation.ts

```diff
@@ -37,9 +37,14 @@ type FailedChangeWithDependencies = {
 const mapObjectDeployErrorToInstance = (error: Error):
 { get: (changeData: ChangeDataType) => string | undefined } => {
   const detectedLanguage = detectLanguage(error.message)
-  const { validationFailed, objectValidationErrorRegex } = multiLanguageErrorDetectors[detectedLanguage]
+  const {
+    validationFailed,
+    objectValidationErrorRegex,
+    settingsValidationErrorRegex,
+  } = multiLanguageErrorDetectors[detectedLanguage]
   const scriptIdToErrorRecord: Record<string, string> = {}
-  const errorMessageChunks = error.message.split(validationFailed)[1]?.split('\n\n') ?? []
+  const splitterRegex = validationFailed.test(error.message) ? validationFailed : settingsValidationErrorRegex
+  const errorMessageChunks = error.message.split(splitterRegex)[1]?.split('\n\n') ?? []
```

Comment:
maybe we should also take all as one chunk in case that `splitterRegex` doesn't find anything
```suggestion
  const errorMessageChunks = error.message.split(splitterRegex)[1]?.split('\n\n') ?? [error.message]
```